v2/internal/controllers/crd_dbformysql_flexibleserver_test.go

```diff
@@ -112,6 +73,37 @@ func Test_DBForMySQL_FlexibleServer_CRUD(t *testing.T) {
 	tc.Expect(exists).To(BeFalse())
 }
 
+func newFlexibleServer(tc *testcommon.KubePerTestContext, owner *genruntime.KnownResourceReference, secretRef genruntime.SecretReference) (*mysql.FlexibleServer, string) {
```

Comment:
minor: I think call the `secretRef` parameter something like `adminPasswordSecretRef` so it's clear what it's going to be used for.

Reply:
Also instead of taking `*genruntime.KnownResourceReference` maybe consider taking a `ResourceGroup` and then calling the `testcommon.AsOwner(rg)` in this helper? That way it's a bit clearer to the caller what the owner actually _is_.